styles.css

```diff
@@ -1 +1,74 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+
+/*Grid mobile*/
+@media screen and (max-width: 800px) {
+    .container {
+        background-color: #fcf0de;
+        display: grid;
+        margin: 0;
+        grid-template-columns: 1fr;
+        grid-template-rows: repeat(7, minmax(50px, auto));
```

Comment:
Wat je hebt gedaan dat werkt. In de uitwerkingen van novi staat grid-template-rows: 50px auto repeat(5, 50px);